tests/pdctl/scheduler/scheduler_test.go

```diff
@@ -74,6 +75,15 @@ func TestScheduler(t *testing.T) {
 		return ""
 	}
 
+	mightExec := func(args []string, v interface{}) {
+		output, err := pdctl.ExecuteCommand(cmd, args...)
+		re.NoError(err)
+		if v == nil {
+			return
+		}
+		json.Unmarshal(output, v)
```

Comment:
Should it ignore the error?

Reply:
Yes, because there could be no result and it returns [500]. We should wait.